sdk/core/System.ClientModel/src/Message/PipelineResponse.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.IO;
+
+namespace System.ClientModel.Primitives;
+
+public abstract class PipelineResponse : IDisposable
+{
+    // TODO(matell): The .NET Framework team plans to add BinaryData.Empty in dotnet/runtime#49670, and we can use it then.
```

Comment:
@KrzysztofCwalina: BinaryData.Empty is already there.
https://github.com/Azure/azure-sdk-for-net/pull/41009#discussion_r1442256737

Reply:
Let me see if I can upgrade the package version then!

Reply:
Looks like the min version needed for System.Memory.Data is 6.0.0 and it brings in a lot of dependencies through System.Text.Json.  @KrzysztofCwalina, would you be ok if we handled this separately?  I'm happy to file an issue to follow up and I know @jsquire would be thrilled to finally update STJ!

Reply:
Tracking with https://github.com/Azure/azure-sdk-for-net/issues/41034